tests/test_customized/TestControllers.py

```diff
@@ -178,3 +178,53 @@ def test_customize_test_creates_with_customize_regression_tests(self, mock_user,
             regression_tests = test.get_customized_regressiontests()
             self.assertIn(2, regression_tests)
             self.assertNotIn(1, regression_tests)
+
+    def test_customize_test_github_server_error(self, mock_user, mock_git, mock_requests):
+        """
+        Test in case github ever returns a 500 error
+        """
+
+        import mod_customized.controllers
+        reload(mod_customized.controllers)
+
+        self.create_user_with_role(
+            self.user.name, self.user.email, self.user.password, Role.tester)
+
+        with self.app.test_client() as c:
+            response_login = c.post('/account/login', 
+                data=self.create_login_form_data(self.user.email, self.user.password))
+
+            response = c.post('/custom/', 
+                data=self.create_customize_form('randomhash', ['linux'],
+                                                            regression_test=[2]), follow_redirects=True)
+
+            # Validate if View Works
+            self.assertEqual(response.status_code, 200)
+            test = Test.query.filter(Test.id == 3).first()
+            self.assertEqual(test, None)
+            self.assertIn("GitHub returned an Error!", str(response.data))
```

Comment:
To reflect to above suggested change 

```suggestion
            self.assertIn("Error contacting Github", str(response.data))
```